docs/install_guides/installing-openvino-apt.md

```diff
@@ -127,22 +130,28 @@ See the example commands below:
 * **Runtime Package**<br>
   On Ubuntu 18.04:
   ```sh
-  sudo apt install intel-openvino-runtime-ubuntu18-2021.1.105
+  sudo apt install intel-openvino-runtime-ubuntu18-2021.4.752
   ```
   On Ubuntu 20.04:
   ```sh
-  sudo apt install intel-openvino-runtime-ubuntu20-2021.1.105
+  sudo apt install intel-openvino-runtime-ubuntu20-2021.4.752
   ```
 * **Developer Package**<br>
   On Ubuntu 18.04:
   ```sh
-  sudo apt install intel-openvino-dev-ubuntu18-2021.1.105 
+  sudo apt install intel-openvino-dev-ubuntu18-2021.4.752 
   ```
   On Ubuntu 20.04:
   ```sh
-  sudo apt install intel-openvino-dev-ubuntu20-2021.1.105
+  sudo apt install intel-openvino-dev-ubuntu20-2021.4.752
   ```
 
+You can also specify the version only, without a specific build number, for example:
+```sh
+apt install intel-openvino-runtime-ubuntu18-2021.4
+```
+This will install the latest build.
```

Comment:
Maybe we can switch the logic? E.g. we recommend to install OV packages without build numbers, but for advanced users we can show how to pick up a dedicated build revision if they want it.

It allows to skip complex logic for most of users who don't care about build revisions.

Reply:
yes, we can. Actually I noticed this option is "To install a specific version", so I added a new option above it as "To install the default version". Also I added `sudo` in the commands. Please check.

Reply:
Do we even need the option for a specific build revision for the LTS version in the main documentation? Users should normally use the latest version. An incompatibility between the latest version and another build is a bug. It seems that older builds are also removed from time to time (the motivation for this PR is that a previously linked version was removed) so it is not wise for users to stick to a particular build number - they cannot count on it being around for a long time.

There's also possible confusion between build and version. "default version" and "specific version" may give the impression it is about, say 2022.2 vs 2021.4.

If needed, we could have a section at the bottom with "advanced options" where we explain this, but it is nice if main documentation can be simple.

Reply:
I agree that we can simplify docs more with removing sections to install a specific version. Let's always suggest latest.
Advanced users might know about ability to select a specific version

Reply:
Updated with "Advance Option" added. Please check.